Tests/ResolverTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the SwaggerResolverBundle package.
+ *
+ * (c) Viktor Linkin <adrenalinkin@gmail.com>
```

Comment:
Для новых файлов наше копирайты.
```suggestion
 * (c) MarfaTech <https://marfa-tech.com>
```